.env.example

```
@@ -1 +1,11 @@
+# Prisma
 DATABASE_URL=
+
+# Better Auth
+BETTER_AUTH_SECRET=p2kG47YycBETeqvinx4wt6NQ50XCAk7m
+BETTER_AUTH_URL=http://localhost:3000 # Base URL of your app
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)